### DISCUSSION TITLE
Quarto absent from R studio

### DISCUSSION THREAD

**Discussion by ghost** [dat-linux/community#55] [FAQ] [answered]
i have been able to install the distro and it works super fine

however, i am still unable to start a quarto project in R studio. As far as i can notice, no option for quarto project or file is showing. I tried many times but i could not figure out how to. A bit odd, because it is a main part of my professional duties to work with quarto. I acknowledge to be a bit lost (coming from windows)

is there any help you could provide ?

**Comment by k1consulting** [comment]
Hi, there's a Quarto installer in the Data Science Extras tab. Re-launch R-Studio after it's installed.

Re Quarto usage in R-Studio, please refer to this; https://quarto.org/docs/tools/rstudio.html

**Comment by ghost** [comment]
Yes, quarto is currently installed and up to date (version 1.5.24) and rbase (version 4.3.3) and rstudio (version 2023.12.1-403)

Nonetheless, no option to create quarto file, nor project is available.

Any clue ?

**Reply by k1consulting** [reply, marked as answer]
![screen](https://github.com/dat-linux/community/assets/114071614/133fa00e-d9f9-4db6-898b-9bb351793705)

**Reply by ghost** [reply]
Problem solved, thanks.

I had to uninstall both app, and reinstall them back. I don't know why it could not link fine before but it works quite well now...

**Reply by dat-linux** [reply]
great!
